### DISCUSSION TITLE
隐藏系统托盘图标的功能

### DISCUSSION THREAD

**Discussion by PJ-568** [JasonWei512/EnergyStarX#47] [Ideas]
### Is it possible to hide the system tray icon?
可否添加一个可选择隐藏系统托盘图标的功能？

这让任务栏更加简洁，也让这个软件离完美更进一步。

**Comment by JasonWei512** [comment]
那样就没法在主窗口不显示时唤起窗口和关闭应用了。托盘图标比较符合一般 Windows 用户的直觉。
不想它显示在任务栏右下角的话，你可以把托盘图标拖到上箭头那里把它隐藏起来。

https://user-images.githubusercontent.com/16899918/219913693-48ac9ca9-6b08-4c4d-847b-52f20619cc99.mp4

**Reply by PJ-568** [reply]
原来如此。我原本的设想是开始菜单的图标可以启动此程序或唤起程序窗口（这样托盘图标就不是刚需）。

![image](https://user-images.githubusercontent.com/63788100/219922466-c74663d0-1228-449f-8869-59da4730bae5.png)
_[Auto Dark Mode就是采用此方案]_

我还是希望您能考虑添加可选择是否隐藏托盘图标的功能，这对小屏多后台的移动用户（特别是像我这样的强迫症）来说还是比较重要的。
谢谢。
:-)

**Reply by JasonWei512** [reply]
点击开始菜单图标唤起窗口以前做过（#10），但是会遇到一个框架的 Bug，所以最后没采用。
